config/routes.rb

```diff
@@ -3,6 +3,10 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  resources :users, only: [:show]
```

Comment:
we chatted about this, but we can refactor all these routes once we have everything built out to use only resources. I'd like to get a bit better with that so maybe I can take it on as a refactor down the line.